frontend/src/components/object/ObjectFilters.vue

```diff
@@ -73,10 +71,7 @@ const metadataValues = computed(() => {
 });
 
 const tagsetValues = computed(() => {
-  // Filter out any tags that don't have an objectID that exist in getUnfilteredObjectIds
-  const filteredVals = getTagSearchResults.value.filter((searchRes) =>
-    getUnfilteredObjectIds.value.some((obj) => obj === searchRes.objectId)
-  );
+  const filteredVals = getTagSearchResults.value;
```

Comment:
I forget why you changed this.. now the filter options shown in the dropdown arent combined with other filtering.